src/app/content/components/Page.tsx

```diff
@@ -13,9 +13,11 @@ export default class PageContent extends Component<PropTypes> {
   public getCleanContent = () => {
     const {content} = this.props;
     return content
+      // remove body and surrounding content
       .replace(/^[\s\S]*<body.*?>|<\/body>[\s\S]*$/g, '')
-      .replace(/<div data-type="document-title">[\s\S]*?<\/div>/g, '')
-      .replace(/<cnx-pi.*>[\s\S]*<\/cnx-pi>/g, '');
+      // fix assorted self closing tags
+      .replace(/<(em|h3|iframe|span|strong|sub|sup|u)([^>]*?)\/>/g, '<$1$2></$1>')
```

Comment:
Did none of the tests change? Would it make sense to add a test for these?

Reply:
....probably

Reply:
can we merge the content refs one before i create more merge conflicts in the page.spec?

Reply:
actually i want some of the utils i made in #48 can we merge that first?